src/__tests__/validator_arrayOf_tests.js

```diff
@@ -0,0 +1,70 @@
+import test from 'ava';
+
+import Validator from '../validator';
+
+test('Validator.validate should flatten structure if arrayOf rule is used', t => {
```

Comment:
I think `'Validator.validate should flatten error structure if array is returned'` better conveys the intent of the test since we want it to work for other rules than arrayOf.